project.md

```
@@ -0,0 +1,138 @@
+                    }
+                }
+            }
+        }).mount('#app');
+    </script>
+</body>
+</html>
+```
+
+### Testing Workflow
+
+1. Start backend server:
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)